src/custom/components/swap/EthFlow/index.tsx

```diff
@@ -68,47 +69,33 @@ const ButtonWrapper = styled.div`
 const MODAL_CLOSE_DELAY = 1000 // 1s
 
 export interface Props {
-  native: Currency
-  wrapped: Token & { logoURI: string }
   nativeInput?: CurrencyAmount<Currency>
-  isNativeIn: boolean
-  isNativeOut: boolean
-  isWrappedIn: boolean
-  isWrappedOut: boolean
-
+  wrapUnrapAmount?: CurrencyAmount<Currency>
   // state
   approvalState: ApprovalState
-  wrapState: WrapType
-
   // modal state
   modalIsOpen: boolean
-
   // cbs
+  openSwapConfirmModalCallback: OpenSwapConfirmModalCallback
   onDismiss: () => void
   approveCallback: (params?: { useModals: boolean }) => Promise<TransactionResponse | undefined>
-  wrapCallback: ((params?: { useModals: boolean }) => Promise<TransactionResponse | undefined>) | undefined
-  swapCallback: (showConfirmModal?: boolean) => void
+  openSwapConfirm(): void
```

Comment:
I'd prefer keeping the "Modal" nomenclature, e.g: `openSwapConfirmModal` etc

Reply:
Got rid of this in the next PR's